fapolicy_analyzer/ui/trust_file_list.py

```diff
@@ -200,7 +225,11 @@ def process_trust(trust, event):
             for data in trust:
                 if event.is_set():
                     return
-                self.__queue.put(self._row_data(data))
+
+                if self.show_trusted or not data.status.lower() == "t":
```

Comment:
This is where the trusted files get skipped from being displayed and the total gets decremented